### DISCUSSION TITLE
Gcode upload to discord before print

### DISCUSSION THREAD

**Discussion by D3v3lish** [cameroncros/OctoPrint-DiscordRemote#211] [Ideas]
Would it be possible to get the dicord bot to upload a thumbnail of gcode, when it starts to print, so i´s easier to see what the project is when printing and uploading to discord underways.?

**Comment by cameroncros** [comment]
Certainly possible. I don't really have time to spend on making such a
change though, So it would be up to you or someone else you can find to
make the change.
